satellite/satellitedb/users.go

```diff
@@ -54,7 +54,11 @@ func (users *users) Insert(ctx context.Context, user *console.User) (_ *console.
 	}
 
 	optional := dbx.User_Create_Fields{
-		ShortName: dbx.User_ShortName(user.ShortName),
+		ShortName:      dbx.User_ShortName(user.ShortName),
```

Comment:
For all these nullable fields, you only want to set them if `IsProfessional` is true.
So you can do something like
```
optional := dbx.User_Create_Fields{
...
    ShortName: dbx.User_ShortName_Null()
...
}
if user.IsProfessional {
    optional.ShortName = dbx.User_ShortName(user.ShortName)
    ...
}
```